org.opent2t.sample.hub.superpopular/com.hue.hub/js/manifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <schemas>
+      <schema id="org.opent2t.sample.hub.superpopular" main="true" type="OCF" />
+      <schema id="oic.core" type="OCF" />
+      <schema id="oic.baseResource" type="OCF" />
+  </schemas>
+  <!-- associated onboarding module -->
+  <onboarding id="org.opent2t.onboarding.huehub">
+    <onboardflow name="getDeveloperInput">
+      <flow>
+        <arg name="client_id" />
+        <description language="en">Ask for Client ID</description>
+        <description language="fr">Demandez Client ID</description>
+      </flow>
+      <flow>
+        <arg name="client_secret" />
+        <description language="en">Ask for Client Secret</description>
+        <description language="fr">Demandez Client Secret</description>
+      </flow>
+    </onboardflow>
+    <onboardflow name="askUserPermission">
+      <flow>
+        <arg name="url" />
+        <description language="en">https://api.meethue.com/oauth2/auth?clientid={client_id}&appid={app_id}&deviceid={device_id}&devicename={device_name}&state={state}&response_type=code</description>
```

Comment:
the & need to be changed to &amp;

without the &amp; it fails syntax test, you can verify using notepad++ and xml tools, verify xml syntax